website/content/docs/commands/pki/issue.mdx

```diff
@@ -0,0 +1,71 @@
+---
+layout: docs
+page_title: pki issue - Command
+description: |-
+  The "pki issue" command issues a new intermediate ("issuer") certificate,
+  using a parent certificate in vault.
+---
+
+# pki issue
+
+This command creates a intermediate certificate authority certificate signed by
+the `<parent>` in the `<child_mount>`, using the options to determine the fields
+on that certificate.
+
+## Usage
+
+Usage: `vault pki issue [flags] <parent> <child_mount> [options]`
+
+- [flags] are optional arguments described below
+
+- `<parent>` is the fully qualified path of the Certificate Authority in vault
+  which will issue the new intermediate certificate.
+
+- `<child_mount>` is the path of the mount in vault where the new issuer is saved.
+
+- [options] are the superset of the options passed to generate/intermediate and
```

Comment:
As a thought, it might be nice to the user to link to the PKI docs on these sections, or mention that you do so below. :-) 

Also, aren't these not "options" but `k=v` tuples?

Reply:
I think options refers to anything that isn't needed for the command to function, and k=v tuples reverses to the form of the option or argument?  So, yes, both?  Let me know if you like the new text better?